apps/studio/components/layouts/ProjectLayout/LayoutHeader/LocalVersionPopover.tsx

```diff
@@ -0,0 +1,157 @@
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="macos">
+                    macOS
+                  </TabsTrigger_Shadcn_>
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="windows">
+                    Windows
+                  </TabsTrigger_Shadcn_>
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="linux">
+                    Linux
+                  </TabsTrigger_Shadcn_>
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="npm">
+                    npm / Bun
+                  </TabsTrigger_Shadcn_>
+                </TabsList_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="macos">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    brew upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="windows">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    scoop upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="linux">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    brew upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="npm">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    npm update supabase --save-dev
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+              </Tabs_Shadcn_>
+            </div>
+          ) : null
+        ) : null}
+
+        <div className="flex flex-col gap-y-2 mt-3">
+          <p className="text-xs text-foreground-lighter">
+            All available image versions of Supabase Studio can be found on our{' '}
+            <InlineLink href="https://hub.docker.com/r/supabase/studio/tags">Docker Hub</InlineLink>
+            .
+          </p>
+        </div>
+
+        <div className="flex items-center gap-x-2 mt-3">
+          <Dialog>
+            <DialogTrigger asChild>
+              <Button type="default">Release schedule</Button>
+            </DialogTrigger>
+            <DialogContent>
+              <DialogHeader className="border-b">
+                <DialogTitle>Stable release schedule</DialogTitle>
+              </DialogHeader>
+              <DialogSection className="flex flex-col gap-y-2">
+                <p className="text-foreground-lighter text-xs font-mono uppercase">
+                  Approximate next release: {approximateNextRelease}
+                </p>
+                <p className="text-sm">
+                  Supabase CLI releases follows a 2 weeks cadence, which stable releases will be
```

Comment:
Supabase CLI releases follow a two-week schedule, with stable updates available through the CLI.
?